utils/UUID_cmp.hh

```diff
@@ -64,6 +78,21 @@ inline uint64_t timeuuid_cmp::uuid_read_lsb(const int8_t* b) noexcept {
            u64(b[14]) << 8  | u64(b[15]);
 }
 
+inline std::strong_ordering timeuuid_cmp::unix_timestamp_tri_compare() noexcept {
+    auto u1 = UUID_gen::get_UUID_from_msb(_o1.begin());
+    auto t1 = UUID_gen::normalized_timestamp(u1).count();
+    auto u2 = UUID_gen::get_UUID_from_msb(_o2.begin());
+    auto t2 = UUID_gen::normalized_timestamp(u2).count();
+    auto res = int64_t_tri_compare(t1, t2);
+    if (res != 0) {
+        return res;
+    }
+    // Compare the random least-significant bytes as unsigned bytes
+    auto lsb1 = bytes_view(_o1.begin() + 8, 8);
+    auto lsb2 = bytes_view(_o2.begin() + 8, 8);
+    return compare_unsigned(lsb1, lsb2);
```

Comment:
This is actually much slower than reading big-endian data and comparing as uint64_t.